services/workflows-service/src/alert/types.ts

```diff
@@ -5,3 +5,33 @@ export type TAuthenticationConfiguration = {
   isValid: boolean;
   webhookSharedSecret: string;
 };
+
+export const BulkStatus = {
+  success: 'success',
+  partial: 'partial',
+  failed: 'failed',
```

Comment:
Let's keep at least the keys consistent with the rest of the system. 🙏🏼 
```suggestion
  SUCCESS: 'success',
  PARTIAL: 'partial',
  FAILED: 'failed',
```